internal/allocator/allocator.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb"
 	"github.com/ydb-platform/ydb-go-genproto/protos/Ydb_Table"
+	"google.golang.org/protobuf/types/known/structpb"
```

Comment:
Keep the order of imported packages. Standard library packages should be imported at first place, then external packages and project internal packages as last one. Refer other files in the projects for examples.